post-processor/googlecompute-export/post-processor.go

```diff
@@ -150,6 +151,10 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packer.Ui, artifact
 			"https://www.googleapis.com/auth/userinfo.email",
 		},
 	}
+	if p.config.ServiceAccountEmail != "" {
+		exporterConfig.ServiceAccountEmail = p.config.ServiceAccountEmail
+	}
+	exporterConfig.CalcTimeout()
```

Comment:
Woops; this one got removed recently

Reply:
```suggestion
```

Reply:
Interesting, I think I rebased it, but it's now fixed.